subprojects/provider/src/test/kotlin/org/gradle/kotlin/dsl/accessors/DefaultPackageTypesTest.kt

```diff
@@ -32,4 +34,29 @@ class DefaultPackageTypesTest {
             equalTo(listOf("Extension"))
         )
     }
+
+    @Test
+    fun `#importsRequiredBy takes container elements into account`() {
```

Comment:
❓  What's the significance of the `#` symbol in the test name?

Reply:
That's meant to signify the name refers to a function (like in javadoc links).

Reply:
Ah, thanks.